siliconcompiler/report/streamlit_report.py

```diff
@@ -5,12 +5,10 @@
 
 try:
     from streamlit.web import bootstrap
+    from streamlit import config as _config
+    _has_streamlit = True
 except ModuleNotFoundError:
```

Comment:
Let's just stick with python's default package management then.
```suggestion
from streamlit.web import bootstrap
from streamlit import config as _config
```

Reply:
this will not work, as a 3.6 package will attempt to import this and fail.

Reply:
Sure then an if statement for the version around it?
I would prefer not to overwrite the "ModuleNotFoundError" because that is very helpful.

Reply:
Okay